interceptors/2.1/_index.md

```diff
@@ -1,26 +1,26 @@
 ---
 title: "Jakarta Interceptors 2.1"
-date: 2022-01-12
+date: 2022-01-26
 summary: "Release for Jakarta EE 10"
 ---
 Jakarta Interceptors defines a means of interposing on business method invocations and specific
 events—such as lifecycle events and timeout events—that occur on instances of Jakarta EE components
 and other managed classes.
 
-The 2.1 release updates dependencies for EE 10 and add a JPMS module-info descriptor.
+The 2.1 release updates dependencies for EE 10 and adds a JPMS module-info descriptor.
 
 * [Jakarta Interceptors 2.1 Release Record](https://projects.eclipse.org/projects/ee4j.interceptors/releases/2.1)
-* [Jakarta Interceptors 2.1 Specification Document](./interceptors-spec-2.1.pdf) (PDF)
-* [Jakarta Interceptors 2.1 Specification Document](./interceptors-spec-2.1.html) (HTML)
+* [Jakarta Interceptors 2.1 Specification Document](./jakarta-interceptors-spec-2.1.pdf) (PDF)
+* [Jakarta Interceptors 2.1 Specification Document](./jakarta-interceptors-spec-2.1.html) (HTML)
 * [Jakarta Interceptors 2.1 Javadoc](./apidocs)
 * [Jakarta EE Platform 10 TCK](TBD) ([sig](TBD),[sha](TBD),[pub](TBD))
```

Comment:
I think this line should state: no TCK delivery for this specification instead of providing an invalid link - TBD